src/import-data/ImportRequirements.test.tsx

```diff
@@ -3,27 +3,23 @@ import React from 'react';
 
 import {
   anvilPfbImportRequests,
-  azureTdrSnapshotImportRequest,
   gcpTdrSnapshotImportRequest,
   protectedGcpTdrSnapshotImportRequest,
 } from './__fixtures__/import-request-fixtures';
 import { ImportRequest } from './import-types';
 import { ImportRequirements } from './ImportRequirements';
 
 describe('ImportRequirements', () => {
-  it.each([
-    { importRequest: azureTdrSnapshotImportRequest, expectedCloudPlatform: 'Microsoft Azure' },
-    { importRequest: gcpTdrSnapshotImportRequest, expectedCloudPlatform: 'Google Cloud Platform' },
-  ] as { importRequest: ImportRequest; expectedCloudPlatform: string }[])(
-    'should show the required cloud platform if any',
-    ({ importRequest, expectedCloudPlatform }) => {
-      // Act
-      render(<ImportRequirements importRequest={importRequest} />);
+  it.each([{ importRequest: gcpTdrSnapshotImportRequest, expectedCloudPlatform: 'Google Cloud Platform' }] as {
```

Comment:
Not terribly important, but since there's only one input here it doesn't need to be a `.each`

Reply:
Fixed: https://github.com/DataBiosphere/terra-ui/pull/5275/commits/369c47bdeaa92675ed4c29ba5bb38e4b470f345c